### DISCUSSION TITLE
License?

### DISCUSSION THREAD

**Discussion by rtali-ia** [camlab-ethz/ConvolutionalNeuralOperator#5] [General]
Hi There,

Have you provisioned a license for this code?

Thanks,
Ronak Tali

**Comment by bogdanraonic3** [comment]
Hi,

There is an NVIDIA license for the CNO2d_original_version. The license is "non-commercial", meaning that the code is for research or evaluation purposes only.

Other codes are not under any other license. However, they may be used for reasearch and evaluation purposes only.

Best,
Bogdan
